webapp/src/routes/Home/FrontLayer/Home/Home.frontLayer.js

```diff
@@ -198,7 +220,7 @@ const HomeFrontLayer = () => {
             <span className="infoLabel"> {t('homeText.text4')}</span>
           </div>
           <div className="boxGroup">
-            <span className="coloredLabel">12,246</span>
+            <span className="coloredLabel">{mentions?.totalMentions || 0}</span>
```

Comment:
Pending to add number string format

Reply:
Also is pending to show this same information related to each exchange, however, the same query is returning that information, I just need the confirmation of the Twitter usernames to make this.